RawEventProcessor.cpp

```diff
@@ -266,13 +267,20 @@ bool RawEventProcessor::process_syscall_event(const Event& event) {
                     num_path += 1;
                     path_recs.emplace_back(rec);
                     if (!path_rec) {
+                        bool isItemZero = false;
+                        unsigned int numNodeFields = 0;
                         for (auto& f: rec) {
                             if (f.FieldName() == SV_ITEM && f.RawValue() == SV_ZERO) {
-                                num_fields += rec.NumFields()-1; // exclude item
+//                                num_fields += rec.NumFields()-1; // exclude item
```

Comment:
Is there any value in leaving this commented line?

Reply:
Yep, removed in update to PR.